src/typings/notifications.ts

```diff
@@ -9,6 +17,7 @@ interface PopUpMessage {
     message: string,
     timeoutMillis: number,
     title: string,
+    button?: PopUpButton,
```

Comment:
how about
```
import {ButtonProps} from "@mui/joy";
```
1. Use `ButtonProps` instead of our own `PopUpButton`.
2. Rename the prop as `primaryAction`. (That way it makes sense for us to add a secondary action button in the future, though not strictly needed for now. )